app/src/main/java/com/mcgearybros/vroomvroom/HtmlContentManager.java

```diff
@@ -18,7 +18,7 @@
     public String currentId;
     public ListIterator<String> iterator;
 
-    public HtmlContentManager(ArrayList<NavigationSubItem> navigationSubItems, LinkedHashMap fullContentHashMap){
+    public HtmlContentManager(ArrayList<NavigationSubItem> navigationSubItems, LinkedHashMap<String, NavigationSubItem> fullContentHashMap){
```

Comment:
Yes, functionally no change, but Android Studio will no longer warn me that the LinkedHashMaps might be different types